test/boolean_test.cpp

```diff
@@ -432,12 +432,10 @@ TEST(Boolean, Precision2) {
 
 TEST(Boolean, DISABLED_SimpleCubeRegression) {
   ManifoldParams().intermediateChecks = true;
-  ManifoldParams().processOverlaps = false;
   Manifold result =
       Manifold::Cube().Rotate(-0.10000000000000001, 0.10000000000000001, -1.) +
       Manifold::Cube() -
       Manifold::Cube().Rotate(-0.10000000000000001, -0.10000000000066571, -1.);
   EXPECT_EQ(result.Status(), Manifold::Error::NoError);
   ManifoldParams().intermediateChecks = false;
-  ManifoldParams().processOverlaps = true;
```

Comment:
What's the trouble with this test?

Reply:
It produces self-intersecting results (more than epsilon):

![image](https://github.com/user-attachments/assets/ff075811-4e8a-4051-9c15-23482e955877)